src/sentry/static/sentry/app/views/onboarding/project/platformCard.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import {flattenedPlatforms} from '../utils';
+import PlatformiconTile from './platformiconTile';
+import classnames from 'classnames';
+
+const PlatformCard = React.createClass({
+  propTypes: {
+    platform: React.PropTypes.string,
+    onClick: React.PropTypes.func
+  },
+
+  render() {
+    const platform = flattenedPlatforms.find(p => p.id === this.props.platform);
```

Comment:
are you objecting to this because it's a linear time lookup? i can preprocess this into a flap map in utils